python/doc/examples/meta_modeling/fields_metamodels/plot_viscous_fall_metamodel.py

```diff
@@ -146,4 +146,4 @@
 ot.ResourceMap.Reload()
 
 # %%
-otv.View.ShowAll()
+ot.View.ShowAll()
```

Comment:
```suggestion
otv.View.ShowAll()
```
the original line was correct, View belongs to the openturns.viewer module, not to openturns directly;

Reply:
yes